### PR TITLE
Add `ssh_{public,private}_key_ssm_path` outputs

### PR DESCRIPTION
**Pull request by rsrchboy**
...as these could be construed as useful :)

### CODE REVIEW

**Comment by osterman** [comment]
Btw, you should totally join our community slack team! This is another great place to ask questions and get quickly unblocked. 

https://slack.cloudposse.com

[Slack Community](https://slack.cloudposse.com)

File: output.tf [hunk 1]
```diff
@@ -2,3 +2,11 @@ output "public_key" {
   value       = "${local.rsa ? join("", tls_private_key.default_rsa.*.public_key_openssh) : join("", tls_private_key.default_ecdsa.*.public_key_openssh)}"
   description = "Contents of the generated public key"
 }
+
+output "ssh_public_key_ssm_path" {
+  value = "${local.ssh_public_key_ssm_path}"
```
Comment (aknysh) [inline output.tf]: Add `description`

File: output.tf [hunk 2]
```diff
@@ -2,3 +2,11 @@ output "public_key" {
...
+}
+
+output "ssh_private_key_ssm_path" {
+  value = "${local.ssh_private_key_ssm_path}"
```
Comment (aknysh) [inline output.tf]: Add `description`

**Comment by rsrchboy** [comment]
Oh, nice -- I didn't realize the README is built that way.  Done.

**Comment by osterman** [comment]
Oops! Looks like we missed this. Yes looks good. Can you rebase or fix the merge conflicts?

**Comment by rsrchboy** [comment]
@osterman done!

**Pull request merged**